examples/categorifier-c-examples.cabal

```diff
@@ -173,6 +173,37 @@ executable recursive-types
     , categorifier-unconcat-integration
   default-language: Haskell2010
 
+executable recursive-types-2
+  hs-source-dirs: recursive-types-2
+  main-is: Main.hs
+  other-modules:
+      F
+  ghc-options:
+      -O0
+      -fexpose-all-unfoldings
+      -fmax-simplifier-iterations=0
+      -fno-ignore-interface-pragmas
+      -fno-omit-interface-pragmas
```

Comment:
We shouldn't need this flag
On Main, right?

Reply:
Not on Main, but `F.hs` needs it since that's where the plugin runs.